docs/_config.yml

```diff
@@ -7,32 +7,45 @@ versions:
 sidebar_categories:
   null:
     - index
-    - example
-    - setup
-    - requests
-    - graphiql
-  Servers:
-    - servers/express
-    - servers/hapi
-    - servers/koa
-    - servers/lambda
-    - servers/micro
-    - servers/restify
-    - servers/azure-functions
-    - servers/adonis
-  Related:
-    - title: Monitoring and caching
-      href: https://www.apollographql.com/docs/engine/setup-node.html
-    - title: graphql-tools
-      href: https://www.apollographql.com/docs/graphql-tools/
-    - title: GraphQL Subscriptions
-      href: https://www.apollographql.com/docs/graphql-subscriptions/
-    - title: Production deployment
-      href: https://dev-blog.apollodata.com/graphql-over-rest-with-node-heroku-and-apollo-engine-fb8581f8d77f
-  Migration:
-    - migration-one-dot
-    - migration-hapi
-    - migration
+    - why-apollo
+    - getting-started
+    - whats-new
+  Essentials:
+    - essentials/building-schema
+    - essentials/queries
+    - essentials/mutations
+  Schema Development:
+    - schemas/types
+    - schemas/resolvers
+    - schemas/context
+    - schemas/organizing-code
+  Running a Server:
+    - server/index
+    - server/middleware
+    - server/queries
+    - server/engine
+    - server/secrets
+  Best Practices:
+    - best-practices/authentication
+    - best-practices/schema-design
```

Comment:
I'll take a stab at this

Reply:
I think making every type nullable should be a best practice, unless an instance of the type cannot exist without the non-null field: https://medium.com/@calebmer/when-to-use-graphql-non-null-fields-4059337f6fc8